flink-contrib/flink-storm-compatibility/flink-storm-compatibility-examples/src/main/java/org/apache/flink/stormcompatibility/util/StormFileSpout.java

```diff
@@ -45,9 +49,16 @@ public StormFileSpout(final String path) {
 	public void open(final Map conf, final TopologyContext context, final SpoutOutputCollector collector) {
 		super.open(conf, context, collector);
 		try {
+			/*  If config is given, it should override constructor path  */
+			if (conf != null && conf.containsKey("textpath")) {
+				this.path = (String)conf.get("textpath");
+			}
+
 			this.reader = new BufferedReader(new FileReader(this.path));
 		} catch (final FileNotFoundException e) {
 			throw new RuntimeException(e);
+		} catch (final NullPointerException e) {
```

Comment:
Why do you catch `NullPointerException`? Can be removed. NPE is a runtime exception anyway and you just re-throw.